### DISCUSSION TITLE
Which are properer in br/ja README, “English"  "简体中文" or their translation in br/ja?

### DISCUSSION THREAD

**Discussion by Octopus058** [fishaudio/fish-speech#570] [Q&A] [answered]
Generally speaking, I believe that in ja README, "简体中文" should not be referred to as "中國語", just like not referring to "Simplified Chinese" in English README. The README of each language should be consistent. What about your views?

**Comment by Octopus058** [comment]
@leng-yue

**Reply by Octopus058** [reply]
So it should be "简体中文" "English" "日本語" consistently, that means they shouldn't be changed with the README language no matter in which README.

**Reply by leng-yue** [reply]
That should be better, do you mind open a PR?

**Reply by Octopus058** [reply, marked as answer]
ok i'll PR it
